numba/tests/test_parallel_backend.py

```diff
@@ -243,6 +254,16 @@ class TestParallelBackendBase(TestCase):
         ]
         all_impls.extend(parfor_impls)
 
+    if config.NUMBA_NUM_THREADS < 2:
+        # Not enough cores
+        masks = []
```

Comment:
For some reason tests still fail when `NUMBA_NUM_THREADS=1` here.

Reply:
Fixed in https://github.com/numba/numba/pull/5044/commits/d787cfb09613ddf0b11ef636d03ae0dcc3db86d1

Reply:
Oh derp!

Reply:
I actually meant for it to be masks = [1] but that fails some tests with `NUMBA_NUM_THREADS=1`. I'm not sure if that's something we should be worried about.

Reply:
It looks like the same tests also fail in master with `NUMBA_NUM_THREADS=1` (they also take a really long time to run).

Reply:
ok, well this fixes it either way then I guess?

Reply:
Yeah this works. I doubt testing set_num_threads(NUMBA_NUM_THREADS) will test very much anyway (beyond what is tested in test_num_threads.py), even when NUMBA_NUM_THREADS=1.